src/inc/corinfo.h

```diff
@@ -1686,6 +1686,8 @@ struct CORINFO_CALL_INFO
     };
 
     CORINFO_CONST_LOOKUP    instParamLookup;    // Used by Ready-to-Run
+
+    BOOL                    secureInvoke;
```

Comment:
> Opinions on which way we should be going?

Is this a per-call property or a per-method property? In other words: for any given method, if we see one secure delegate invoke, will we see only secure delegate invokes, or will we see a mix? If it is a per-call property, then we should probably use a `flags` field; if it is a per-method property, then a per-method flag seems more appropriate.

> It is unfortunate to have to another 4 bytes in this structure for this rare case

It doesn't look like this type has been aggressively packed: for example, I see a `CORINFO_CONTEXT_HANDLE` above followed by a `BOOL`. Perhaps we should consider adjusting this struct's layout for better packing, even if we do not end up needing to add a `flags` or `secureInvoke` field. cc @pkukol